src/bin/cql-stress-cassandra-stress/settings/option/rate.rs

```diff
@@ -116,13 +124,13 @@ fn prepare_parser() -> (ParamsParser, RateParamHandles) {
     let throttle = parser.simple_param(
         "throttle=",
         None,
-        "throttle operations per second across all clients to a maximum rate (or less) with no implied schedule",
+        "limits the rate of operations per second with a schedule; does not fix the displayed latencies in regards to coordinated-omission problem (i.e. end_time - actual_start_time)",
```

Comment:
```suggestion
        "limits the rate of operations per second with a schedule; see `fixed` option if coordinated-omission is needed",
```

it's a bit weird to me, to express with such details what it's not gonna do :)
but just point to where is can be explain with more

Reply:
Thanks, I wasn't sure how to phrase it properly. But I'd still be more specific: ``see `fixed` option if latency adjustment to coordinated-omission is needed"``. Coordinated-omission is not needed - what user might need is the latency correction if C-O appears.

Reply:
fixed